MultigridProjector/Logic/MultigridProjection.cs

```diff
@@ -1450,7 +1451,11 @@ public BuildCheckResult CanBuild(MySlimBlock projectedBlock, bool checkHavokInte
                 return BuildCheckResult.OK;
 
             var gridPlacementSettings = new MyGridPlacementSettings { SnapMode = SnapMode.OneFreeAxis };
-            if (MyCubeGrid.TestPlacementAreaCube(builtGrid, ref gridPlacementSettings, min, max, previewBlock.Orientation, previewBlock.BlockDefinition, ignoredEntity: builtGrid, isProjected: true))
+
+            if (checkHavokIntersections && CheckVoxels(previewBlock))
```

Comment:
Redundant condition. The `!checkHavokIntersections` condition on 1450 already returns if `checkHavokIntersections` is `false`.